Mobile Buy SDK Sample Apps/Mobile Buy SDK tvOS Sample/Mobile Buy SDK tvOS Sample/PageViewController.swift

```diff
@@ -0,0 +1,115 @@
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        self.dataSource = self
+        
+        let initialViewController = productImageViewController(forPage: 0)
+        setViewControllers([initialViewController], direction: .forward, animated: false, completion: nil)
+    }
+    
+    // MARK: UIPageViewControllerDataSource
+    
+    func pageViewController(_ pageViewController: UIPageViewController, viewControllerBefore viewController: UIViewController) -> UIViewController? {
+        let index = indexOfImageItem(forViewController: viewController)
+        
+        if index > 0 {
+            return productImageViewController(forPage: index - 1)
+        }
+        else {
+            return nil
+        }
+    }
+    
+    func pageViewController(_ pageViewController: UIPageViewController, viewControllerAfter viewController: UIViewController) -> UIViewController? {
+        let index = indexOfImageItem(forViewController: viewController)
+        
+        if index < productImages.count - 1 {
+            return productImageViewController(forPage: index + 1)
+        }
+        else {
+            return nil
+        }
+    }
+    
+    func presentationCount(for pageViewController: UIPageViewController) -> Int {
+        return productImages.count
+    }
+    
+    func presentationIndex(for pageViewController: UIPageViewController) -> Int {
+        guard let currentViewController = pageViewController.viewControllers?.first else { fatalError("Unable to get the page controller's current view controller.") }
+        
+        return indexOfImageItem(forViewController: currentViewController)
+    }
+    
+    // MARK: Convenience
+    
+    private func indexOfImageItem(forViewController viewController: UIViewController) -> Int {
+        guard let viewController = viewController as? ProductImageViewController else { fatalError("Unexpected view controller type in page view controller.") }
+        guard let viewControllerIndex = productImages.index(of: viewController.imageItem.imageLink) else { fatalError("View controller's image item not found.") }
+        
+        return viewControllerIndex
+    }
+    
+    private func productImageViewController(forPage index: Int) -> ProductImageViewController {
+        let image = productImages[index]
+        
+        if let cachedController = productImageViewControllerCache.object(forKey: image.identifier.stringValue as NSString) {
+            return cachedController
+        } else {
+            let storyboard: UIStoryboard = UIStoryboard(name: "Main", bundle: nil)
+            guard let controller = storyboard.instantiateViewController(withIdentifier: ProductImageViewController.identifier) as? ProductImageViewController
+                else { fatalError("Unable to instantiate a ProductImageViewController.") }
+            controller.loadView()
```

Comment:
`loadView()` should never be called directly. Use `view` to force the view to load.

However, why do you need the view to load?

Should loading the view be implicit in the `configure(imageItem:)` method, or the `productImageView` property getter?